### DISCUSSION TITLE
Why does this line in FloatingArrow set height of the svg equal to the width?

### DISCUSSION THREAD

**Discussion by tylersayshi** [floating-ui/floating-ui#2494] [General]
https://github.com/floating-ui/floating-ui/blob/6b12fdef95d7b09f276816ce9aead601be55204d/packages/react/src/components/FloatingArrow.tsx#L135

**Comment by atomiks** [comment]
The arrow element size needs to be a square, so the inner shapes are bounded by that constraint.
